services/repository/src/test/java/org/sagebionetworks/repo/web/controller/TeamControllerAutowiredTest.java

```diff
@@ -64,4 +68,27 @@ public void testListTeams() throws Exception {
 	public void testGetTeamIdsByMember() throws Exception {
 		servletTestHelper.getTeamIdsByMember(dispatchServlet, 1L, TeamSortOrder.TEAM_NAME, true);
 	}
+
+	/**
+	 * PLFM-6390
+	 * @throws Exception
+	 */
+	@Test
```

Comment:
Let us upgrade to Junit 5, make sure the class extends AbstractAutowiredControllerJunit5TestBase